Comp2025/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -306,10 +306,11 @@ private void configureButtonBindings( )
     //
     // Operator - POV buttons
     //
-    m_operatorPad.pov(0).onTrue(m_elevator.getMoveToPositionCommand(m_elevator::getElevatorStowed));
-    m_operatorPad.pov(90).onTrue(new LogCommand("operPad", "POV 90"));
-    m_operatorPad.pov(180).onTrue(m_elevator.getMoveToPositionCommand(m_elevator::getElevatorCoralL1));
-    m_operatorPad.pov(270).onTrue(m_elevator.getMoveToPositionCommand(m_elevator::getElevatorCoralL4));
+
```

Comment:
While it's OK to hook some positions to the POV buttons right now, this is not going to suffice for competition. We'll need more automation or we'll run out of buttons for all the possible elevator heights. Again, OK for now, but we'll need to develop the rest of the code.

Reply:
When someone posts a comment or review, either a) change the code, or b) absorb the info in the comment (this case). Then, in both cases resolve the conversation once you have received the message.